w3lib/url.py

```diff
@@ -68,18 +68,34 @@ def safe_url_string(url, encoding='utf8', path_encoding='utf8'):
 
     # quote() in Python2 return type follows input type;
     # quote() in Python3 always returns Unicode (native str)
-    return urlunsplit((
-        to_native_str(parts.scheme),
-        to_native_str(netloc).rstrip(':'),
-
-        # default encoding for path component SHOULD be UTF-8
-        quote(to_bytes(parts.path, path_encoding), _safe_chars),
-
-        # encoding of query and fragment follows page encoding
-        # or form-charset (if known and passed)
-        quote(to_bytes(parts.query, encoding), _safe_chars),
-        quote(to_bytes(parts.fragment, encoding), _safe_chars),
-    ))
+    
+    if bool(escape_URL):
+        return urlunsplit((
+            to_native_str(parts.scheme),
+            to_native_str(netloc).rstrip(':'),
+
+            # default encoding for path component SHOULD be UTF-8
+            to_native_str(parts.path),
```

Comment:
I'm not sure if the name ``escape_URL`` matches what is happening here - if ``escape_URL`` is True, we're not quoting the path, but if it's False we do quote it. To me something like ``quote_path`` would make more sense (with appropriate inverted logic).

Reply:
Thanks for reviewing, would ```encode_path``` be a good name?

Reply:
Because we are encoding ```"``` to ```%22``` in URL